src/Symfony/Component/JsonEncoder/Attribute/JsonEncodable.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonEncoder\Attribute;
+
+/**
+ * Mark a class as encodable.
```

Comment:
```suggestion
 * Marks a class as encodable.
```

I'm also wondering if the description is useful at all tbh

Reply:
I think we can remove it yes.